RELEASE_NOTES

```diff
@@ -3,9 +3,33 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
-The latest version of easybuild-easyconfig provides 4,819 easyconfig files, for 778 different software packages,
+The latest version of easybuild-easyconfig provides 5,093 easyconfig files, for 799 different software packages,
 34 different (compiler) toolchains, 12 software bundles and 1 meta-package.
 
+v2.6.0 (January 26th 2016)
+--------------------------
+
+feature + bugfix release
+- add easyconfigs for foss/2016a and intel/2016 common toolchains (#2310, #2311, #2339, #2363)
+    incl. easyconfigs for Boost, CMake, Python, Perl using these toolchains
+- added example easyconfig files for 21 new software packages:
+    BLASR (#922), BioKanga (#2247), BoltzTraP (#2365), basemap (#2221), CppUnit (#2271), EggLib (#2335),
+    FLASH (#2281), GLM (#2288), hub (#2249), MACS2 (#1983), MotEvo (#843), numba (#2243), PGI (#1833, #2367),
+    PLY (#2305), PaStiX (#2319, #2326), patchelf (#2327), pip (#2284), RSEM (#2316), RcppArmadillo (#2289),
```

Comment:
first PIL easyconfig was added in v2.4.0, cfr. http://easybuild.readthedocs.org/en/latest/Release_notes.html#v2-4-0-november-10th-2015